metrics/scaling/k8s_scale_rapid.sh

```diff
@@ -211,8 +211,8 @@ help()
 	usage=$(cat << EOF
 Usage: $0 [-h] [options]
    Description:
-	Launch a series of workloads and take memory metric measurements after
-	each launch.
+	Launch a series of workloads and take memory metric measurements using collectd
```

Comment:
a little suggestion: using single quotes to refer another command/application e.g. `collectd` to avoid confusion with a typo.

Reply:
Done.